src/GraphQL/Types/DateGraphType.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Globalization;
+using Newtonsoft.Json.Linq;
+
+namespace GraphQL.Types
+{
+    public class DateGraphType : ScalarGraphType
```

Comment:
Agree. Sorry, I've already got this part sorted in my local branch. We're using this in our own code base, and was a quickfix I had to put in place recently that I've later refactored. Will get my most recent changes in here (hopefully tomorrow), and make sure to include that one.